openedx/core/djangoapps/edx_global_analytics/utils/utilities.py

```diff
@@ -79,46 +79,17 @@ def get_previous_month_start_and_end_dates():
 
 def get_coordinates_by_ip():
     """
-    Gather coordinates by server IP address with FreeGeoIP service.
+    Gather coordinates by server IP address with ip-api service.
```

Comment:
Title in the multiple doc strings should be separated from the other text by the empty line.